src/sentry/integrations/vercel/client.py

```diff
@@ -66,3 +73,10 @@ def create_env_variable(self, vercel_project_id, key, value):
         data = {"key": key, "value": value, "target": "production"}
         response = self.post(self.ENV_VAR_URL % vercel_project_id, data=data)
         return response
+
+    def update_env_variable(self, vercel_project_id, key, value):
+        self.delete(
+            self.DELETE_ENV_VAR_URL % (vercel_project_id, key, "production"), allow_text=True
```

Comment:
@ceorourke I think we should have a specific `delete_env_variable` function that we can call to be consistent with the rest of what we're doing here.